cmd/ops/deprecation.go

```diff
@@ -10,16 +10,18 @@ import (
 )
 
 const (
-	schemaFileFlag = "schema"
+	schemaFileFlag  = "schema"
+	xcodeFormatFlag = "xcode"
 )
 
 var (
 	//Flags
-	schemaFile string
+	schemaFile  string
+	xcodeFormat bool
 
 	//Command
 	deprecationsCmd = &cobra.Command{
-		Use:   "deprecation [flags] queries_directory",
+		Use:   "deprecation [flags] queries_directory|queries_list_file",
```

Comment:
what type of file does `queries_list_file` stand for?

Reply:
I will update the help to make it clearer